base/file.jl

```diff
@@ -641,14 +641,28 @@ function chmod(path::AbstractString, mode::Integer; recursive::Bool=false)
     nothing
 end
 
+sentinel_code(i::Integer) = i
+sentinel_code(s::Symbol) =
+    if s == :preserve
+        -1
+    elseif s == :remove
+        -2
```

Comment:
This appears to only be true on a few operating systems:
http://pig.made-it.com/uidgid.html
https://en.wikipedia.org/wiki/User_identifier#Special_values

Reply:
I was under the impression that this value as a sentinel was a feature of `libuv`.

Reply:
This is why we can't have nice things

Reply:
Is setting the user/group to "no one" even an option on all systems?

Reply:
As always with unix, "it depends". Most uid numbers aren't mapped to an actual user (simple reason: many implementations allow for 2^32 numbers, but only usually there's only a few hundred users at most). So chances are pretty good that a number picked at random will not map to an existing user. It's unclear whether you were asking about mapping to the explicitly reserved "not an account" account "nobody" (or perhaps "nfsnobody", or for NTAUTH "NULL SID"), or whether it's possible to pick a number that doesn't map to an account.